packages/schema/src/Codegen/files/generated/types.ts

```diff
@@ -110,9 +113,21 @@ export class TypesFile extends File {
         : any
 
       ${body}
+      
+      ${Object.values(this.codegen.schema.types)
+        .filter(type => type.kind === 'ENUM')
+        .map(
+          type =>
+            `${this.generateComments(
+              this.schemaTypeComments(type)
+            )}export enum ${type.name} { \n
+            ${(type as SchemaEnumType).enumValues.map(k => `${k} = '${k}' \n`)}
+          }`
+        )
+        .join('\n')}
 
       ${Object.values(this.codegen.schema.types)
-        .filter(type => type.kind !== 'INPUT_OBJECT')
+        .filter(type => !['INPUT_OBJECT', 'ENUM'].includes(type.kind))
```

Comment:
can you keep this how it was, and just check for type.kind in the .map below?

```ts
          type =>
            `${this.generateComments(
              this.schemaTypeComments(type)
            )}${type.kind === 'enum' ? `export enum ${type.name} { \n
            ${(type as SchemaEnumType).enumValues.map(k => `${k} = '${k}' \n`)}
          }` : ORIG}`
```

Reply:
Changed.